src/app/ErrorView/ErrorView.tsx

```diff
@@ -70,9 +72,11 @@ export const ErrorView: React.FunctionComponent<ErrorViewProps> = (props) => {
               <StackItem>{props.message}</StackItem>
               {props.retry && (
                 <StackItem>
-                  <Button variant="link" onClick={props.retry}>
-                    Retry
-                  </Button>
+                  {props.retryButtonEnabled || (
```

Comment:
`props.retry` is optional so if it is specified, that means retry is enabled so we can remove `props.retryButtonEnabled`?

Reply:
Please also remove this `Text` component here: 

https://github.com/cryostatio/cryostat-web/blob/47b7c85cc20cb28e629e024f9bfd8f5dd0e91e90/src/app/ErrorView/ErrorView.tsx#L83

I dont think we need it. Not sure why it was there even.